lib/srv/sess.go

```diff
@@ -334,8 +341,8 @@ func (r *sessionRegistry) PartyForConnection(sconn *ssh.ServerConn) *party {
 
 	for _, session := range r.sessions {
 		session.Lock()
+		defer session.Unlock()
```

Comment:
JFYI - unlock will be done when you return from the function, not at the end of the loop scope - not sure if you want this, the previous version looked more efficient